src/OSPSuite.Presentation/Presenters/Importer/ImporterPresenter.cs

```diff
@@ -126,7 +124,14 @@ public void SetDataSource(string dataSourceFileName)
 
       public void ImportData(object sender, EventArgs e)
       {
-         OnTriggerImport.Invoke(this, new ImportTriggeredEventArgs { DataSource = _dataSource });
+         try
+         {
+            OnTriggerImport.Invoke(this, new ImportTriggeredEventArgs { DataSource = _dataSource });
+         }
+         catch (InconsistenMoleculeAndMoleWeightException exception)
```

Comment:
This is quite suprising here. You raise a event and you know that a listener will raise a specific error...🤔

Reply:
I am not sure how this should change.....should the listener actually take care of handling the exception right? so they are not coupled together?